src/core/config.go

```diff
@@ -596,7 +596,10 @@ type Configuration struct {
 	Alias            map[string]*Alias  `help:"Allows defining alias replacements with more detail than the [aliases] section. Otherwise follows the same process, i.e. performs replacements of command strings."`
 	Plugin           map[string]*Plugin `help:"Used to define configuration for a Please plugin."`
 	PluginDefinition struct {
-		Name string `help:"The name of the plugin"`
+		Name          string   `help:"The name of the plugin"`
+		Description   string   `help:"A description of what the plugin does"`
+		BuildDefsDir  []string `help:"Directory to look in when prompted for help topics that aren't known internally" example:"build_defs"`
+		Documentation string   `help:"A link to the documentation for this plugin"`
```

Comment:
nit: might be confused with the literal documentation. Perhaps `DocumentationSite` would be better?